web/cadence/scripts/get_account_items.cdc

```diff
@@ -0,0 +1,10 @@
+  import NonFungibleToken from 0xNonFungibleToken
```

Comment:
Indentation is a bit off in some of the Cadence files

Reply:
good point, fixing